cognite/powerops/resync/v2/shop_to_assets.py

```diff
@@ -0,0 +1,349 @@
+# Mypy does not understand the pydantic classes that allows both alias and name to be used in population
+# type: ignore # noqa: [call-arg]
```

Comment:
I would still suggest including a link to the open issue so that people can easily check the status as this mypy disable should be removed whenever a fix has been made